community/kernel/src/main/java/org/neo4j/kernel/impl/api/OperationsFacade.java

```diff
@@ -1541,6 +1541,7 @@ public void unregisterExecutingQuery( ExecutingQuery executingQuery )
             BasicContext ctx = new BasicContext();
             ctx.put( Context.KERNEL_TRANSACTION, tx );
```

Comment:
`currentTransaction` is only available inside GraphDatabaseFacade. Builtin procedures that need access to kernel transaction depend on it being in the context.